pkg/reconciler/openshift/openshift.go

```diff
@@ -133,6 +133,28 @@ func getArgsForRedhatHaRedisInitContainer() []string {
 // policyRulesForClusterConfig defines rules for cluster config.
 func policyRulesForClusterConfig() []rbacv1.PolicyRule {
 	return []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{
+				"*",
```

Comment:
With this, we can get rid of `"compliance.openshift.io",` rule

Reply:
Removed this rule !! Thanks!